### ISSUE TITLE
Image attributes stripped in editor

### ISSUE THREAD

**Issue by nathbailey** [ayush013/ngx-quill-upload#6]
If an image has any attributes other than "src", "class" and "style" this is removed when rendered in the editor. After investigating the code, this appears to be caused by the following lines of code in "ngx-quill-upload\__ivy_ngcc__\fesm2015\ngx-quill-upload.js"

```
const blotPath = 'blots/block/embed';
const BlockEmbed = Quill.import(blotPath);
class ImageBlot extends BlockEmbed {
```
Changing the blotPath variable to "formats/image" resolves this issue.

**Comment by ayush013** [comment]
Hi, 

Quill uses classes by default. To override this behaviour we have to extend Image Blot to add support for custom atttributes.

This will be supported in next release.

**Comment by ayush013** [comment]
Will be fixed once #7 is merged
